pkg/network/ebpf/c/protocols/protocol-dispatcher-helpers.h

```diff
@@ -12,6 +12,10 @@ static __always_inline bool is_tcp_termination(skb_info_t *skb_info) {
     return skb_info->tcp_flags & (TCPHDR_FIN | TCPHDR_RST);
 }
 
+static __always_inline bool is_tcp_ack(skb_info_t *skb_info) {
+    return skb_info->tcp_flags == TCPHDR_ACK;
```

Comment:
I think `return skb_info->tcp_flags&TCPHDR_ACK` may cover a wider range of scenarios we may be interested in?

Reply:
no! the data is being sent as PSH | ACK
so we must check for "ACK only"